pulp_container/app/tasks/sync_stages.py

```diff
@@ -498,23 +501,6 @@ def relate_blob(self, dc):
         related_dc = dc.extra_data.get("blob_relation")
         return BlobManifest(manifest=related_dc.content, manifest_blob=dc.content)
 
-    def relate_manifest_tag(self, dc):
-        """
-        Relate an ImageManifest to a Tag.
-
-        Args:
-            dc (pulpcore.plugin.stages.DeclarativeContent): dc for a Tag
-
-        """
-        related_dc = dc.extra_data.get("man_relation")
-        dc.content.tagged_manifest = related_dc.content
-        try:
-            dc.content.save()
```

Comment:
This is particularly bad, as content should be immutable. And i believe it is the root cause of https://pulp.plan.io/issues/9292 .